src/fortran_routing/diffusive_v02frwk/maindiff_v2.f90

```diff
@@ -0,0 +1,1474 @@
+module mdiffv2
+
+    implicit none
+
+    double precision, dimension(:,:), allocatable :: celty_g, diffty_g, q_g, elv_g, qpx_g
+    double precision, dimension(:), allocatable :: cel_av_g
+    double precision, dimension(:), allocatable :: qlatj_g
+    double precision, dimension(:), allocatable :: eei_g, ffi_g, exi_g, fxi_g
+    double precision, dimension(:), allocatable :: celty_m_g, diffty_m_g, q_m_g, qpx_m_g
+    double precision, dimension(:), allocatable :: qlatj_m_g
+    double precision, dimension(:), allocatable :: eei_m_g, ffi_m_g, exi_m_g, fxi_m_g
+    double precision, dimension(:), allocatable :: dx_ar_m_g
+    double precision, dimension(:,:,:), allocatable :: bfuf_g    
+    double precision:: z_g, bo_g, traps_g, tw_g, twcc_g, So_g, mann_g, manncc_g
+
+
+contains
+    !*--------------------------------------------------------------------------------
+    !*       Route network by using Python-to-Fortran network traversal map together
+    !*          with diffusive routing engines( test )
+    !
+    !*--------------------------------------------------------------------------------
+    subroutine diffnw(dtini_g, t0_g, tfin_g, saveinterval_g, saveinterval_ev_g, dt_ql_g, dt_ub_g, dt_db_g, &
```

Comment:
Both the fortran modules here suffer from a lack of intuitive function names and documentation.  There seems to be some consistence semantics in parameter naming, e.g. `_g` and `_ar` or `_ev`, but there is no documentation to help explain what that convention means.  At a minimum, each function should have a good comment that describes the semantics of the function/subroutine, what its inputs are (in a human readable description of each parameter) and what its outputs are.

There are also significant portions of commented out code.  If that code has some meaning that is relevant to keep in the repository, then add a comment or two describing why it is there but isn't used.  If it doesn't have merit, remove it.

Reply:
@kumdonoaa - As a part of ongoing scientific development of the diffusive wave model, please keep these comments in mind.

Reply:
@hellkite500 @awlostowski-noaa - Thanks for the comment. The Fortran code is still a test version so lots of functions and variables come and go. A final version should have clean and useful documentation.

Reply:
@kumdonoaa Sounds good.  I have gone back and forth on development strategies, but I have found that including that documentation during the development is often more helpful than adding it after the fact.  Often times I find that writing out the summaries and documenting the inputs/outputs help me better design and implement.  Just some food for thought!

Reply:
@hellkite500 I can't agree more.  An updated fortran module I will git push very soon with additional functions will have such documentation.

Reply:
New updates to `diffusive_utils.py`:
- docstrings!
- cleanout unused libraries